WordPress/src/main/java/org/wordpress/android/util/config/RemoteConfig.kt

```diff
@@ -0,0 +1,42 @@
+package org.wordpress.android.util.config
+
+import com.google.android.gms.tasks.Task
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings.Builder
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.R
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T.UTILS
+import javax.inject.Inject
+
+/**
+ * Do not use this class outside of this package. Use [AppConfig] instead
```

Comment:
@jd-alexander This might be a good candidate for a lint-rule.

Reply:
Thanks for spotting this @malinajirka I appreciate it.